### DISCUSSION TITLE
NUC980(NK980IIoT) board support for booting using SD Card.

### DISCUSSION THREAD

**Discussion by Nachiketha0509** [OpenNuvoton/NUC980-linux-4.4.y#140] [Q&A]
Hi,
We are using the NK980IIoT board for a particular application and we are trying to boot it using an SD Card. After doing that we are trying to see the same using serial terminal. We have built the Linux 4.4.115+ version of the linux kernel and we wanted to carry out further tests by using some commands.

1) One particular command we need is the 'touch' command and when we enter this command it says touch applet not found. How do we resolve this issue? 

2)Also even when we connect the 'Ethernet cable' it is still not being connected to the net nor is it reflecting on the router web page.

Kindly help us if you know the resolution to the same
Thank you

**Comment by ychuang3** [comment]
1. 'touch' command is supported by busybox, which is the default 'init system' selected by NUC980 buildroot. Do you use buildroot to build NUC980 linux-4.4 kernel?  Please refer to https://github.com/OpenNuvoton/NUC970_Buildroot. 
2. The same answer as 1.

**Reply by Nachiketha0509** [reply]
This NUC970 buildroot may have a problem and when I enter the make command its giving me errors. Please help me resolve this

**Comment by ychuang3** [comment]
Do you select the configuration first?

```
# make nuvoton_nuc980_iot_defconfig
# make
```

You can find available configurations in the 'configs' folder.
